Build/Scripts/SystemCenter.buildproj

```diff
@@ -92,6 +92,11 @@
       <GitServer Condition="'$(GitServer)' == ''">git@github.com:GridProtectionAlliance/SystemCenter.git</GitServer>
     </PropertyGroup>
   </Target>
+  <Target Name="AfterUpdateRepository">
+      <Message Text="Getting latest updates from submodule..."/>
+      <Exec Command="%22$(GitClient)%22 submodule update --recursive --remote" WorkingDirectory="$(LocalFolder)" Condition="'$(SetUpWorkspace)' != 'False'"/>
```

Comment:
Should this be done in `UpdateDependencies.bat`? Also, our nightly build script uses `SetUpWorkspace=False`.

```bat
ECHO Getting latest...
SET SetUpWorkspace=False
START /B /D "%LocalFolder%\" /WAIT git gc
START /B /D "%LocalFolder%\" /WAIT git fetch
START /B /D "%LocalFolder%\" /WAIT git reset --hard origin/master
START /B /D "%LocalFolder%\" /WAIT git clean -f -d -x

...

ECHO Building SystemCenter NightlyBuild version...
SET DoNotPush=%InheritedDoNotPush%
CALL CALLBAT "%LocalFolder%\Build\Scripts\" BuildNightly.bat SystemCenter.output
```

Reply:
Per our conversation this is not actually a dependency update but part of the `pull` process. 
We will get this added to `UpdateDependecies.bat`, `SC-BN.bat` and `SC-BB.bat` as well for completeness. In the future it may be worth adding it to GSF,